<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPayment</class>
 <widget class="QDialog" name="CPayment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>205</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Оплата наличными</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblAmount">
     <property name="text">
      <string>Сумма по счёту</string>
     </property>
     <property name="buddy">
      <cstring>edtAmount</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="edtAmount">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="prefix">
      <string notr="true"/>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="maximum">
      <double>9999999.990000000223517</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblFee">
     <property name="text">
      <string>&amp;Внесено</string>
     </property>
     <property name="buddy">
      <cstring>edtFee</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="edtFee">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="maximum">
      <double>9999999.990000000223517</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblChange">
     <property name="text">
      <string>Сдача</string>
     </property>
     <property name="buddy">
      <cstring>edtChange</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="edtChange">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="prefix">
      <string notr="true"/>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>-9999999.990000000223517</double>
     </property>
     <property name="maximum">
      <double>9999999.990000000223517</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edtAmount</tabstop>
  <tabstop>edtFee</tabstop>
  <tabstop>edtChange</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CPayment</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CPayment</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
